### PR TITLE
Shared content fields for Virtual Y entities

### PR DESCRIPTION
**Pull request by anpolimus**
Steps:

- [ ] Enable openy_gc_shared_content module
- [ ] Expect Share and Origin fields at Videos, Blog Posts, Series entities forms

### CODE REVIEW

File: modules/openy_gc_shared_content/config/update/core.entity_form_display.eventinstance.live_stream.default.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    - field.field.eventinstance.live_stream.field_ls_share
+    - field.field.eventinstance.live_stream.field_ls_title
+    - recurring_events.eventinstance_type.live_stream
+  module:
+    - datetime_range
+    - entity_browser
+    - link
+    - text
+_core:
+  default_config_hash: IvQoBEruRdxDlWwUTE3UOnUd2RXefw1dOTfJL-e5fKQ
```
Comment (podarok) [inline modules/openy_gc_shared_content/config/update/core.entity_form_display.eventinstance.live_stream.default.yml:23]: _core and hash should be removed...

File: modules/openy_gc_shared_content/config/update/core.entity_form_display.eventinstance.virtual_meeting.default.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    - field.field.eventinstance.virtual_meeting.field_ls_title
+    - field.field.eventinstance.virtual_meeting.field_vm_link
+    - recurring_events.eventinstance_type.virtual_meeting
+  module:
+    - datetime_range
+    - entity_browser
+    - link
+    - text
+_core:
+  default_config_hash: Awd9v-CMayuQ7BYPtDZW1f_wkvwr2JhayVVjvb0zN9k
```
Comment (podarok) [inline modules/openy_gc_shared_content/config/update/core.entity_form_display.eventinstance.virtual_meeting.default.yml:23]: Same

File: modules/openy_gc_shared_content/config/update/core.entity_form_display.eventseries.live_stream.default.yml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    - field.field.eventseries.live_stream.field_ls_share
+    - recurring_events.eventseries_type.live_stream
+  module:
+    - datetime_range
+    - entity_browser
+    - link
+    - recurring_events
+    - text
+_core:
+  default_config_hash: BbReucrdBBzHhGqVC8YU5a56JmPiO57cjL3MZlD98hs
```
Comment (podarok) [inline modules/openy_gc_shared_content/config/update/core.entity_form_display.eventseries.live_stream.default.yml:22]: Same

File: modules/openy_gc_shared_content/config/update/core.entity_form_display.node.vy_blog_post.default.yml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    - field.field.node.vy_blog_post.field_vy_blog_image
+    - node.type.vy_blog_post
+  module:
+    - entity_browser
+    - link
+    - path
+    - scheduler
+    - text
+_core:
+  default_config_hash: HzIihqB_DEV2HH16X50NklpEtM8fz6sN8XxppsI4dEY
```
Comment (podarok) [inline modules/openy_gc_shared_content/config/update/core.entity_form_display.node.vy_blog_post.default.yml:18]: Everywhere

**Comment by hamrant** [comment]
New version in this PR - https://github.com/fivejars/openy_gated_content/pull/75

**Pull request closed without merging**